### PR TITLE
MR-7 Added steps to install MinGw c/c++ complier to the README.md

### PR DESCRIPTION
**Pull request by BrahmpreetSingh07**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,24 @@ An RTS version of the classic game: minesweeper.
 
 # Dependencies
 
-- C compiler (specify in Makefile)
+- C compiler (specify in Makefile) 
 - Raylib ([https://www.raylib.com/])
 - Make
+    
+Windows setup:[link]
```
Comment (Porkius) [inline README.md]: What is the [link] specifying?

**Review by Porkius — CHANGES_REQUESTED** [review]
Good work! The only issue I see is on line 11, which uses a placeholder "link" name. Other than that I believe it is good to merge.

**Review by Porkius — APPROVED** [review]
Looks good! I think it's ready to be merged!

**Comment by SuchithSridhar** [comment]
Looks good, merging.

**Review by SuchithSridhar — APPROVED** [review]
Looks good, merging.

**Pull request merged**